com.unity.render-pipelines.high-definition/Documentation~/Compositor-User-Guide.md

```diff
@@ -49,3 +49,7 @@ The Render Schedule is a re-orderable list of Composition Layers and Sub-layers.
 ## Composition Parameters
 This section shows every exposed property that is not an input Composition Layer (for example, a Float to control the brightness of the final composition or a Color to tint a Texture2D). In this section, the window allows you to edit each property value outside of the Composition Graph. It is good practice to expose properties from the graph to the Graphics Compositor window, instead of hard-coding their values. This helps you to share composition profiles between Projects because those you do not need to open the Composition Graph to edit any values.
 
+## Composition of screen-space UI
+The Graphics Compositor can be used to compose screen-space canvas (UI) elements. However, in this case the camera sub-layer that draws the UI:
```

Comment:
You can use the Graphics Compositor to compose screen-space canvas (UI) elements. However, in this case, the camera sub-layer that draws the UI: